docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,182 @@
+    - name: gh-wh-receiver
+      github:
+        secretRef:
+          name: gh-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the configuration. We can obtain 
+this URL using the following command:
+
+```shell
+  kubectl \
+    get projectconfigs \
+    kargo-demo \
+    -n kargo-demo \
+    -o=jsonpath='{.status.webhookReceivers}'
+```
+
+
+## Registering with Github
+
+When configuring on Github, you can configure either a webhook or an app. We will outline instructions for both, starting with webhooks.
+
+### Webhooks
+
+1. Navigating to the Webhooks Dashboard
+
+    1. Navigate to <Hlt>Settings</Hlt>
+    1. Click <Hlt>Webhooks</Hlt>
+    1. Click <Hlt>Add webhook</Hlt>
+
+![Step 1](./img/webhooks/123.png "Settings")
+
+2. Registering the Webhook
+
+    1. Set the <Hlt>Payload URL</Hlt> to the value we retrieved from the 
+    [Retrieving the Receiver's URL](#retrieving-the-receivers-url) step.
+    1. Set the <Hlt>Content type</Hlt> field to `application/json`.
+    1. Set the <Hlt>Secret</Hlt> field to the value we assigned the `secret` key 
+    from the [Configuring the Receiver](#configuring-the-receiver) step.
+
+![Step 3](./img/webhooks/4.png "Add Webhook")
+
+:::note
+Leave the <Hlt>Just the push event</Hlt> field checked unless you're
+looking to subscribe to `ghcr` events. If you're looking to subscribe to `ghcr` events you should select <Hlt>Let me select individual events</Hlt> and then 
+select <Hlt>Packages</Hlt>. This requires that you have connected the repository
+and package. For more information on connecting repositories and packages refer to the 
+[Github Docs here](https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package).
+:::
+
+![Step 5](./img/webhooks/5.png "Event Subscription")
+  
+Finally, confirm <Hlt>Active</Hlt> is toggled and click <Hlt>Add webhook</Hlt>.
+
+![Step 6](./img/webhooks/6.png "Submit Form")
+
+3. Verifying Connectivity
+
+From your Webhooks Dashboard, click on the webhook you just created.
+
+![Step 7](./img/webhooks/7.png "Created")
+
+Navigate to <Hlt>Recent Deliveries</Hlt>.
+
+![Step 8](./img/webhooks/8.png "Recent Deliveries")
+
+Click on the <Hlt>ping</Hlt> event and ensure a successful response was returned.
+
+![Step 9](./img/webhooks/9.png "Response")
+
+
+### Apps
+
+It may be tedious to configure webhooks for each of your Github repositories. 
+You can instead opt to configure a 
+[Github App](https://docs.github.com/en/apps); allowing you to receive events 
+from all or select repositories.
+
+1. Navigate to the Github Apps Dashboard
+
+    1. Navigate to <Hlt>Settings</Hlt>. This will be listed in a dropdown menu that is toggled by clicking your Github avatar.
+    1. Click <Hlt>Developer Settings</Hlt>.This will be in the bottom left-hand corner of the settings dashboard.
+    1. Click <Hlt>Github Apps</Hlt>.
+
+![Step 1](./img/apps/123.png "Settings")
+
+2. Register a new Github App
```

Comment:
I recommend using `1.` for every item in an ordered list. It spares us from having to re-number everything every time an item is added to the list or removed from it.

```suggestion
1. Register a new Github App
```